### PR TITLE
global-box: update naming and add template

### PR DESCRIPTION
**Pull request by alexkilgour**
- add handlebars templates
- update naming to meet current conventions

### CODE REVIEW

File: packages/global-box/view/box.hbs [hunk 1]
```diff
@@ -0,0 +1 @@
+<div class="c-box">{{content}}</div>
```
Comment (joeyciechanowicz) [inline packages/global-box/view/box.hbs]: Does content want to be escaped?

Alternatively, do we also want to use partials that allow content for this sort of thing?
```
// Template
<div class="c-box">
    {{> @partial-block }}
</div>

// Usage
{#> box }}
  My Content
{{/box}}


``

Comment (alexkilgour) [reply packages/global-box/view/box.hbs]: yeah, could do that. let me take a look.

Comment (joeyciechanowicz) [reply packages/global-box/view/box.hbs]: For simple components (like this one) one could support both methods as well.

```
{{#if content}}
<div class="c-box">
   {{content}}
</div>
{{#else}}
<div class="c-box">
    {{> @partial-block }}
</div>
{{/if}}
```

Comment (alexkilgour) [reply packages/global-box/view/box.hbs]: could do that, but this is just an example implementation, so I think an example of how `@partial-block` works should suffice, which I have now added 🎉

**Pull request merged**